legend-engine-xt-persistence-pure/src/main/resources/core_persistence/persistence/metamodel.pure

```diff
@@ -602,4 +602,26 @@ Class meta::pure::persistence::metamodel::TestData
 Class meta::pure::persistence::metamodel::ConnectionTestData
 {
   data      : meta::pure::data::EmbeddedData[1];
+}
+
+Class meta::pure::persistence::metamodel::AllRowsEquivalentToJson extends meta::pure::test::assertion::TestAssertion
```

Comment:
Same comments as in protocol classes.
Also if we add these lets update the daigram to reflect this too

Reply:
Sure, added the diagram